Makefile

```diff
@@ -46,7 +46,7 @@ RELEASE_DIR := $(CURDIR)/$(BUILDDIR)/artifacts
 ifeq ("$(TELEPORT_DEBUG)","true")
 BUILDFLAGS ?= $(ADDFLAGS) -gcflags=all="-N -l"
 else
-BUILDFLAGS ?= $(ADDFLAGS) -ldflags '-w -s $(KUBECTL_SETVERSION)' -trimpath
+BUILDFLAGS ?= $(ADDFLAGS) -ldflags '-w -s $(KUBECTL_SETVERSION)' -trimpath -buildmode=pie
```

Comment:
Is `pie` build mode commonly available for all the target we build objects to?
IIRC, Go didn't support `pie` on all targets

Reply:
Yeah, seems pretty commonly supported.

https://github.com/golang/go/blob/35222eeb78ddb84f5787729c5f149cfd73d63ef8/src/internal/platform/supported.go#L187-L198

I could hardcode that list, but seems fairly well-supported? Is there something in that list that is missing?

Reply:
Yes, it seems complete for our use case.

For some reason I tried to figure out the modes in the docs, but as always, the best document is always the code itself.